addon/templates/components/cf-field/input/radio.hbs

```diff
@@ -1,17 +1,19 @@
-{{#each choices as |optionEdge i|}}
-  {{#let optionEdge.node as |option|}}
-    {{#if (gt i 0)}}<br>{{/if}}
-    <label class={{if field.isInvalid "uk-form-danger"}}>
-      <input
-        class="uk-radio uk-margin-small-right"
-        type="radio"
-        name={{field.pk}}
-        value={{option.slug}}
-        checked={{eq option.slug field.answer.value}}
-        disabled={{disabled}}
-        onchange={{action onSave option.slug}}
-      >
+{{#each field.options as |option i|}}
+  {{#if (gt i 0)}}<br>{{/if}}
+  <label class={{if field.isInvalid "uk-form-danger"}}>
+    <input
+      class="uk-radio uk-margin-small-right"
+      type="radio"
+      name={{field.pk}}
+      value={{option.slug}}
+      checked={{eq option.slug field.answer.value}}
+      disabled={{disabled}}
+      onchange={{action onSave option.slug}}
+    >
+    {{#if option.disabled}}
+      <strike class="uk-text-muted" title={{t "caluma.form.optionNotAvailable"}}>{{option.label}}</strike>
```

Comment:
This element is obsolete.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strike

Reply:
Perfekt!